index.d.ts

```diff
@@ -0,0 +1,17 @@
+export const info: string;
+
+/**
+@example
+```
+import logSymbols = require('log-symbols');
+
+console.log(logSymbols.success, 'Finished successfully!');
+// On good OSes:  ✔ Finished successfully!
+// On Windows:    √ Finished successfully!
+```
+*/
+export const success: string;
+
+export const warning: string;
+
+export const error: string;
```

Comment:
Sorry for being unclear, I meant that these should not be named export. They don't mean anything without the context.